app/src/main/java/com/github/sdpcoachme/messaging/ChatActivity.kt

```diff
@@ -109,6 +109,13 @@ class ChatActivity : ComponentActivity() {
 
         store = (application as CoachMeApplication).store
         emailFuture = store.getCurrentEmail()
+            .exceptionally {
```

Comment:
Yes, this is the code we need for the bugfix! Also, the comment makes it clear what exactly the bug was about and how it is fixed.